go.mod

```diff
@@ -357,6 +357,8 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
 
+require cloud.google.com/go/kms v1.4.0
```

Comment:
Why does this get its own require statement instead of being in the block with the rest?

Reply:
yeah, some tool put it there, thanks for pointing it out, fixed